dynawo/sources/Launcher/launcher.in

```diff
@@ -31,7 +31,7 @@ DYNAWO=${BIN_DIR}/dynawo
 GENERATE_PREASSEMBLED=${SBIN_DIR}/generate-preassembled
 COMPILEMODELICAMODEL=${SBIN_DIR}/compileModelicaModel
 DUMPMODELBIN=${SBIN_DIR}/dumpModel
-COMPILELIBMODELICAOMCBIN=${SBIN_DIR}/compileLibModelicaOMC
+COMPILEMODELICAMODELINLIBBIN=${SBIN_DIR}/compileModelicaModelInLib
```

Comment:
I would propose compileDynawoModelicaModelInDynamicLib or compileCppModelicaModelInDynamicLib

Reply:
I changed in compileCppModelicaModelInDynamicLib